plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogConnectorSmokeTest.java

```diff
@@ -133,6 +134,36 @@ public void testRenameSchema()
                 .hasStackTraceContaining("renameNamespace is not supported for Iceberg Glue catalogs");
     }
 
+    @Test
```

Comment:
`@Override` ?

Reply:
Why did we need to override the test ? Please add a code comment as well

Reply:
It's not override.

Reply:
Got it, why don't we move testCommentView from BaseConnectorTest to BaseConnectorSmokeTest or move this code to BaseConnectorSmokeTest ?